app-load-params-db.json

```diff
@@ -303,6 +303,18 @@
     "curve": ["ed25519"],
     "path": ["44'/799'", "44'/354'"]
   },
+  "PEN": {
+    "appFlags": {"nanos": "0x000", "nanos2": "0x000", "nanox": "0x200", "stax": "0x200"},
```

Comment:
`nanos `should be removed here ?

Reply:
For nanos there are two versions, regular and XL. For the other devices only one

Reply:
@tdejoigny_ledger